### PR TITLE
Add Russian translations

### PR DESCRIPTION
**Pull request by illright**
Issue: #584 

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

- [x]  issue was opened to discuss proposed changes before starting implementation.
- [ ]  during development, `node` version specified in `package.json` was used (ie using [nvm](https://github.com/creationix/nvm)).
- [x]  package versions and package-lock.json were not changed (`npm install --no-save`).
- [x]  app version number was not changed.
- [x]  all new code has tests to ensure against regressions.
- [ ] `npm run lint` reports no offenses.
- [ ] `npm run test` is error-free.
- [ ]  README and CHANGELOG were updated accordingly.

### Description of the Change

Added Russian translations. In some places there are pseudo-comments, hoping you could shed some light on the context of some strings to be able to translate them more accurately.

### CODE REVIEW

**Comment by illright** [comment]
As a suggestion for string keys like `seconds_0`, maybe it'd be helpful to clarify the range of numbers that they cover, something like `seconds_[5; +inf)` (though this might seem clumsy and non-conformant to the identifier regex)

**Comment by hovancik** [comment]
thanks @illright 

the format there is a standard from the libary and is looks different for different languages. Ie English has `name` and `name_plural`, Chinese has no plural, etc. 

You can put your language code here and see ranges: https://jsfiddle.net/sm9wgLze

I'll add this to original issue so other can see it as well.

File: app/locales/ru.json [hunk 1]
```diff
@@ -1,193 +1,193 @@
...
-    "microbreakIn": "Mini Break in $t(utils.seconds, {'count': {{seconds}} })",
-    "breakIn": "Long Break in $t(utils.seconds, {'count': {{seconds}} })",
-    "resumingBreaks": "Работа возобновлена",
+    "microbreakIn": "Мини-перерыв через $t(utils.seconds, {'count': {{seconds}} })",
+    "breakIn": "Длинный перерыв через $t(utils.seconds, {'count': {{seconds}} })",
+    "resumingBreaks": "Продолжаем перерывы",
     "downloadLatestVersion": "Загрузить последнюю версию",
-    "toBreak": "Long Break",
-    "toMicrobreak": "Mini Break",
+    "toBreak": "длинному перерыву",
```
Comment (illright) [inline app/locales/ru.json:8]: Lowercased and assuming continuation of `skipToTheNext`

File: app/locales/ru.json [hunk 2]
```diff
@@ -1,193 +1,193 @@
...
+    "restoreDefaults": "Восстановить настройки по умолчанию",
+    "warning": "Это действие вернет настройки к первоначальным, ваши настройки будут утеряны.",
+    "continue": "Продолжить",
+    "cancel": "Отменить",
     "preferences": "Настройки"
   },
   "break": {
-    "postpone": "Postpone this break",
-    "skip": "Skip this break",
+    "postpone": "Отложить",
```
Comment (illright) [inline app/locales/ru.json:27]: Assuming that the context implies the word "break"

File: app/locales/ru.json [hunk 3]
```diff
@@ -1,193 +1,193 @@
...
     "nav": {
-      "settings": "Settings",
-      "schedule": "Schedule",
-      "theme": "Theme",
-      "about": "About",
+      "settings": "Настройки",
+      "schedule": "Расписание",
+      "theme": "Тема",
+      "about": "О программе",
       "heart": "Love Stretchly"
```
Comment (illright) [inline app/locales/ru.json:38]: Not sure what is meant here. Sounds cool, but could you clarify the intent of this option? :)

File: app/locales/ru.json [hunk 4]
```diff
@@ -1,193 +1,193 @@
...
-      "fullscreen": "Full screen",
-      "showIdeas": "Show exercise tips during breaks",
-      "allScreens": "Shows breaks on all monitors",
-      "monitorIdleTime": "Monitor system idle time (countdown is paused if system is idle)",
-      "monitorDnd": "Show breaks even in Do Not Disturb mode",
-      "language": "Select langauge:",
-      "restoreDefaults": "Restore defaults"
+      "openAtLogin": "Запускать Stretchly автоматически при входе в систему",
+      "showBreaksIn": "Показывать перерывы:",
+      "window": "в окне",
```
Comment (illright) [inline app/locales/ru.json:43]: Lowercased and assuming context of `showBreaksIn`

File: app/locales/ru.json [hunk 5]
```diff
@@ -1,193 +1,193 @@
...
+      "restoreDefaults": "Восстановить настройки по умолчанию"
     },
     "schedule": {
-      "miniBreaks": "Mini Breaks:",
-      "miniBreaksInfo": "Mini Breaks are short breaks taken regularly to give you a chance to stretch and relax.",
-      "enableMiniBreaks": "Enable Mini Breaks",
+      "miniBreaks": "Мини-перерывы:",
+      "miniBreaksInfo": "Мини-перерывы это регулярные короткие перерывы, чтоб размяться и отдохнуть.",
+      "enableMiniBreaks": "Включить мини-перерывы",
       "breakFor": "Break for:",
```
Comment (illright) [inline app/locales/ru.json:56]: What is meant here and in the next one?

File: app/locales/ru.json [hunk 6]
```diff
@@ -1,193 +1,193 @@
...
+      "sounds": "Звуки:",
+      "enableSounds": "Включить звуки",
       "crystalGlass": "хрусталь",
       "windChime": "ветер",
       "ticToc": "тик так",
-      "menubarIcon": "Menubar icon:",
-      "colour": "Colour",
-      "monochrome": "Monochrome",
-      "invertedMonochrome": "Inverted Monochrome"
+      "menubarIcon": "Иконка в трее:",
```
Comment (illright) [inline app/locales/ru.json:83]: Assuming that menubar == tray

File: app/locales/ru.json [hunk 7]
```diff
@@ -1,193 +1,193 @@
...
+      "colour": "Цветная",
```
Comment (illright) [inline app/locales/ru.json:84]: The following 3 keys are assuming the context of the `menubarIcon`

File: app/locales/ru.json [hunk 8]
```diff
@@ -1,193 +1,193 @@
...
-      "version": "Version ",
-      "latestVersion": "Latest version ",
-      "notifyNewVersion":"Automatically notify on app updates",
-      "learnMore": "To learn more about Stretchly features, view a tutorial, download the latest version or contact us for support, ",
-      "ourWebsite": "visit our website",
+      "version": "Версия ",
+      "latestVersion": "Последняя версия ",
+      "notifyNewVersion": "Уведомлять о наличии обновлений",
+      "learnMore": "Чтоб узнать больше о функционале Stretchly, пройдите обучение, скачайте последнюю версию или свяжитесь с нами для получения поддержки, ",
+      "ourWebsite": "посетив наш сайт",
```
Comment (illright) [inline app/locales/ru.json:94]: Assuming continuation of `learnMore`

File: app/locales/ru.json [hunk 9]
```diff
@@ -1,193 +1,193 @@
...
+      "alreadyContributor": "Я уже участник",
+      "authenticateUsing": "Войти с помощью:",
+      "contributorPreferences": "Настройки участников"
     }
   },
   "contributorPreferences": {
-    "title": "Contributor Preferences",
-    "notifications": "Break notifications:",
-    "notificationsInfo": "System notification shown before breaks to let you prepare for it.",
+    "title": "Настройки участников",
```
Comment (illright) [inline app/locales/ru.json:110]: Here and in some other places, the title case was omitted, hope that's okay

File: app/locales/ru.json [hunk 10]
```diff
@@ -1,193 +1,193 @@
...
+    "notifications": "Уведомления о перерывах:",
+    "notificationsInfo": "Системные уведомления, показвающиеся перед перерывами, чтоб дать время приготовиться.",
     "beforeLongBreak": "Long Break:",
```
Comment (illright) [inline app/locales/ru.json:113]: Unsure about the context here and in the next one

File: app/locales/ru.json [hunk 11]
```diff
@@ -1,193 +1,193 @@
...
+    "customIdeasInfo": "Позволяет настроить свои собственные идеи для перерывов в конфигурационном файле (для продвинутых пользователей).",
+    "useIdeasFromSettings": "Использовать идеи упражнений из конфигурационного файла",
+    "breakPostpone": "Возможность откладывать перерывы:",
+    "breakPostponeInfo": "Если этот пункт включен в основных настройках, вы можете откладывать перерывы. Последующие настройки позволяют указать, когда, на какой промежуток временни и сколько раз можно откладывать перерывы.",
+    "miniBreaks": "Мини-перерывы:",
+    "postponableFor": "Можно отложить в течение:",
+    "postponeFor": "Откладывать на:",
+    "maxPostpones": "Сколько раз можно откладывать:",
+    "longBreaks": "Длинные перерывы:",
+    "pauseUntilMoring": "Приостановить до утра:",
```
Comment (illright) [inline app/locales/ru.json:127]: Not sure why there's a colon here. 
Are the following 3 entries all part of one form field?

File: app/locales/ru.json [hunk 12]
```diff
@@ -1,193 +1,193 @@
...
-    "indefinitely": "Indefinitely",
-    "dndMode": "Do Not Disturb is on",
-    "nextLongBreak": "Next Long Break",
-    "nextMiniBreak": "Next Mini Break",
-    "afterMiniBreak_0": "after {{count}} Mini Break",
-    "afterMiniBreak_1": "after {{count}} Mini Breaks",
-    "afterMiniBreak_2": "after {{count}} Mini Breaks",
-    "resuming": "Resuming"
+    "paused": "Приостановлено",
+    "indefinitely": "на неопределённый срок",
```
Comment (illright) [inline app/locales/ru.json:184]: Lowercased

File: app/locales/ru.json (hunk 1 above)
Comment (hovancik) [reply app/locales/ru.json:8]: Yep, it's item in Tray menu, Skip to next, Long/Mini break

File: app/locales/ru.json (hunk 2 above)
Comment (hovancik) [reply app/locales/ru.json:27]: This is inside the actual Break Window, the bottom part 

image

File: app/locales/ru.json (hunk 3 above)
Comment (hovancik) [reply app/locales/ru.json:38]: It's this screen: 

image

For example, in Slovak, I translate it as "Help Stretchly"

File: app/locales/ru.json (hunk 4 above)
Comment (hovancik) [reply app/locales/ru.json:43]: image

Comment (hovancik) [reply app/locales/ru.json:43]: I guess if it's lowercase depends on Grammar of your langauge

File: app/locales/ru.json (hunk 5 above)
Comment (hovancik) [reply app/locales/ru.json:56]: Here's the pic, it's the settings for Mini breaks

image

**Comment by hovancik** [comment]
@illright Would it be possible for you to run Stretchly locally and see where the translations are or should I keep addings screenshots to your comments? 

The best for testing should be if you could run it yourself locally

**Comment by hovancik** [comment]
:+1: 

If you go to main.js, and change the line `isContributor: false` to `isContributor: true`, on next start of the app you will be able to Click on button "Contributor Preferences" in "Love Stretchly" page and open extra settings. 

If you need help, I am on gitter https://gitter.im/stretchly/Lobby

**Comment by illright** [comment]
When I clicked "Sign in with GitHub" in the "Love Stretchly" screen, I got this screen, the strings for which I have not seen in the locale file. Also it has a typo in the word "feeling" :p
image

**Comment by illright** [comment]
Also this is confusing to me both in English and in Russian:
image
I'd expect the units of "postponable for" to be time, not percents. Perhaps this label could be rephrased

**Comment by illright** [comment]
image
This space before the full stop is also unfortunate

**Comment by illright** [comment]
And it would be really nice to be able to translate break ideas as well

**Comment by hovancik** [comment]
@illright thanks ;] 

1. the non-translatable login is auth webserver, there are no translations, I've fixed the typo ;] 
2. `postponable for` is percentage of break in which you can postpone. `Postpone for` is how long it will be postponed. Feel free to use text that you think will make sense, ie in first explanation paragraph. 
3. interesting, not sure why there is white space before full stop, should not be. Will have to look into it. There's basically `span`, `a`, `span` so should not be any space
4. Hopefully breaks will be translatable soon, I am applying for RGSoC where students should work on it. 

Let me know if I can merge or you wanna make some change :)

**Comment by illright** [comment]
So with this last commit I've translated "percent" as "percent of duration of the break". This should make enough sense, but then I'd recommend making the same adjustment for other languages, because, as I said, it didn't make sense to me even in English up until you explained it.

Other than that, yep, good to merge

**Pull request merged**